design/one-pager-resource-connectivity-mvp.md

```diff
@@ -410,10 +411,9 @@ could be implemented in one of two ways:
    order to determine that it is named `externalname` when submitting requests
    to the GCP API.
 
-This document suggests the first approach, both because it requires less logic
-and fewer Kubernetes API calls to implement, and because it enables Crossplane
+This document uses a combination of these approaches. For GCP case approach one is used both because it requires less logic and fewer Kubernetes API calls to implement, and because it enables Crossplane
```

Comment:
I've wrapped the rest of this document at 80 characters, which I personally feel makes the markdown easier to read in its raw text form. I don't feel super strongly about keeping the wrap, but I do think we should avoid mixing styles within the same document.

Reply:
@negz what tool do you use for wrapping the lines to 80 chars?